fluent/fluent.go

```diff
@@ -569,6 +570,32 @@ func (n *nextHopEntry) WithNextHopNetworkInstance(ni string) *nextHopEntry {
 	return n
 }
 
+// DecapsulateHeader represents the enumerated set of headers that can be decapsulated
+// from a packet.
+type DecapsulateHeader int64
+
+const (
+	_ DecapsulateHeader = iota
+	// IPinIP specifies that the header to be decpsulated is an IPv4 header, and is typically
+	// used when IP-in-IP tunnels are created.
+	IPinIP
```

Comment:
Shouldn't "in" be capitalized? As in `IPInIP`? Otherwise it may read like `I-Pin-IP`.

By the way, my PR #95 used IPInIP, so let me know if you do prefer IPinIP, and I'll fix it there.

Reply:
I think canonically, we write this as `IP-in-IP` outside of code, which converted to `IPinIP` here. Sometimes we call this `IPIP`. I prefer `IPinIP` just as that is the muscle memory for folks writing about rfc1853.

Reply:
Okay, let me change my PR #95 to use IPinIP.